README.md

```diff
@@ -4,22 +4,25 @@
 
 Common mathematical graph algorithms implemented in PHP
 
->   You're viewing the contents of the `master` development brach. Note that this
-    branch is subject to active development and will contain breaking changes
-    for the upcoming release. If you want to use the latest release version,
-    see also the `v0.8.x` release branch for more details.
-
->   Note: This project is in beta stage! Feel free to report any issues you encounter.
+> **Development version:** This branch contains the code for the upcoming 0.9 release.
+> For the code of the current stable 0.8 release, check out the
+> [`0.8.x` branch](https://github.com/graphp/algorithms/tree/0.8.x).
+>
+> The upcoming 0.9 release will be the way forward for this package.
+> However, we will still actively support 0.8.x for those not yet
+> on the latest version.
+> See also [installation instructions](#install) for more details.
```

Comment:
```suggestion
> **Development version:** This branch contains the code for the upcoming `v0.9` release.
> For the code of the current stable `v0.8` release, check out the
> [`0.8.x` branch](https://github.com/graphp/algorithms/tree/0.8.x).
>
> The upcoming `v0.9` release will be the way forward for this package.
> However, we will still actively support `0.8.x` for those not yet
> on the latest version.
> See also [installation instructions](#install) for more details.
```
I think we should name the versions as suggested. We could also go ahead and name them `v0.9.0` and `v0.8.2` instead of `v0.9` and `v0.8`, but I don't think there's much gained and we don't know for sure if `v0.8.2` will be the last `v0.8` release.

We could also merge the current state of your PR as is, this way it would be compatible to all other projects with the same section.

Not sure which one is really "better" here, I leave this decision up to you :)

Reply:
Updated to consistently write out "upcoming version 0.9" instead of "upcoming 0.9" or "upcoming `v0.9`". WDYT? :+1: